### PR TITLE
traffic_light_yunrong

### PR DESCRIPTION
**Pull request by guyunrong**

### CODE REVIEW

**Comment by nsuitter** [comment]
@guyunrong I don't see an assignment from you submitted on Moodle. Could you submit one please so we can assign you a grade? Thanks!

**Comment by guyunrong** [comment]
Hi Professor Suitter:

```
I have revised my submission on Moodle into :https://github.com/miis-l10n/website-l10n_homework/pull/88

?I have no idea why it doesn't work last time. Can you see it now?
```

**Comment by nsuitter** [comment]
@guyunrong I'm still not seeing it. Have you checked out this post from Bruno http://ilearn.miis.edu/mod/forum/discuss.php?d=19316

**Comment by brossi1** [comment]
@guyunrong I think when you insert the URL as a comment and save, you have to go in again and check the "this is my own work" box and hit save a second time.

**Comment by guyunrong** [comment]
Hi Bruno:

```
?I have tried for another time. Is it Ok now?
```

File: Almond/traffic_light/traffic_light.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+var Stop=document.querySelector(".Stop")
+	var Slow=document.querySelector(".Slow")
```
Comment (nsuitter) [inline Almond/traffic_light/traffic_light.js:2]: From this line down, you can leave this indented similar to line 1

File: Almond/traffic_light/traffic_light.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<html>
+<head>
+	<title>Traffic Light</title>
+	<link rel="stylesheet" type="text/css" href="traffic_light.css">
+
+
+
+</head>
+<body>
+<div class="words">
```
Comment (nsuitter) [inline Almond/traffic_light/traffic_light.html:11]: The Div tag should be indented in one level from the body tag

File: Almond/traffic_light/traffic_light.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<ul>
```
Comment (nsuitter) [inline Almond/traffic_light/traffic_light.html:12]: The ul tag should be indented in one level from the div tag

**Comment by nsuitter** [comment]
@guyunrong nice work on the homework. You need to be a bit more mindful of indentation but it looks great other than that. Nice work on the js!

:+1:

**Pull request merged**